signer/src/blocklist_client.rs

```diff
@@ -44,11 +44,19 @@ impl BlocklistClient {
     pub fn new(ctx: &impl Context) -> Option<Self> {
         let config = ctx.config().blocklist_client.as_ref()?;
 
-        let config = Configuration {
+        let mut config = Configuration {
             base_path: config.endpoint.to_string(),
             ..Default::default()
         };
 
+        // Url::parse defaults `path` to `/` even if the parsed url was without the trailing `/`
```

Comment:
I was just wondering who I had to blame for this behavior, stdlib that thinks it is smarter than us!